COMETwebapp/Components/Viewer/ViewerBody.razor.cs

```diff
@@ -68,19 +66,23 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
 
             if (firstRender)
             {
-                await base.OnInitializedAsync();
-                this.ViewModel.InitializeViewModel();

```

Comment:
Why remove this line of code?

Reply:
The VM is initalized when setting the Iteration